deployment/grafana/README.md

```diff
@@ -0,0 +1,573 @@
+# Grafana Helm Chart
```

Comment:
It is found that this page is almost the same with [this page](https://artifacthub.io/packages/helm/grafana/grafana/6.29.2).
If it is the same, please just give a link somewhere, no need to copy all the content to here.

Reply:
I have deleted some original files of Grafana and update `user-guide-for-observability-management.md`. Readers can read this markdown file to learn how to install Grafana with our sedna's dashboards and specific `values.yaml`.